library/globals.inc.php

```diff
@@ -2967,10 +2967,10 @@ function gblTimeZones()
     'Connectors' => array(
 
         'site_addr_oath' => array(
-            xl('Site Address (required for OAuth2 and FHIR)'),
+            xl('Site Address (required for OAuth2, FHIR and CCDA)'),
             'text',
-            '',
-            xl('Site Address (required for OAuth2 and FHIR). Example is') . ' https://localhost:8300 .'
+            'https://' . $_SERVER['HTTP_HOST'] . $GLOBALS['webroot'],
```

Comment:
good idea in trying to figure it out automatically (and then user can always manually fix it if needed). btw, will this capture port?

Reply:
don't think need to put webroot in this though, since that is added on when using this in global in codebase (done this way I recall since don't always need the webroot like in the oauth stuff)

Reply:
actually didn't mean for that to be there. wanted to wait until I found a home for the resolveHost() routine and use for this.
you're right, don't need webroot. I was playing.